src/subject_name/verify.rs

```diff
@@ -31,29 +31,25 @@ pub(crate) fn verify_cert_dns_name(
 ) -> Result<(), Error> {
     let cert = cert.inner();
     let dns_name = untrusted::Input::from(dns_name.as_ref().as_bytes());
-    NameIterator::new(
-        Some(cert.subject),
-        cert.subject_alt_name,
-        SubjectCommonNameContents::Ignore,
-    )
-    .find_map(|result| {
-        let name = match result {
-            Ok(name) => name,
-            Err(err) => return Some(Err(err)),
-        };
+    NameIterator::new(Some(cert.subject), cert.subject_alt_name)
+        .find_map(|result| {
+            let name = match result {
+                Ok(name) => name,
+                Err(err) => return Some(Err(err)),
```

Comment:
Not totally sure what to do about the Codecov warning on this line (and on the one later in this method) --- it's saying this line was "added", but that's only because the change to remove the `SubjectCommonNameContents` from the call to `NameIterator::new` was removed, resulting in `rustfmt` reformatting all this code. This exact match statement was there before, and (presumably) also not covered by tests?

Reply:
We're not super nit-picky about this kind of codecov finding IME. I think you make a reasonable case for why it isn't important to address in this branch.